### DISCUSSION TITLE
replacement(not found by me, found by coolelectronics, sharpjack, and  sprinkzmc on tn discord)

### DISCUSSION THREAD

**Discussion by frogmasamune** [3kh0/ext-remover#335] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
![• Discord  #kajigs  TitaniumNetwork](https://user-images.githubusercontent.com/117880221/207975748-4ec60c82-d69b-47a7-8b97-fd7d7e799312.png)

Credit goes to CoolElectronics#4683, SprinkzMC#8421, and Sharp_Jack#4374 on discord 

bookmarklet is 

javascript: let shim = false;var ids = prompt("extension ids (comma separated)").split(",");setInterval(()=>{ids.forEach((id)=> opener.chrome.developerPrivate.updateExtensionConfiguration({extensionId: id, fileAccess: shim}));shim = !shim;}, 250);

**Comment by 3kh0** [comment]
Alr i will add this, thanks!

**Reply by Aka-but-nice** [reply]
I just did this and sadly It doesn't work but I think I understand how this works, basically, when you enter the id of the extension it reloads the extension over and over, so when you go to like 1v1.lol the extension can't block it fast enough and YOU can continue winning every game without a mf teacher doing anything about it
